finch/default.cfg

```diff
@@ -15,7 +15,8 @@ maxprocesses = 10
 parallelprocesses = 2
 
 [finch]
-bccaqv2_url = https://boreas.ouranos.ca/thredds/catalog/birdhouse/pcic/BCCAQv2/catalog.xml
+default_dataset = bccaqv2
+dataset_bccaqv2 = https://boreas.ouranos.ca/thredds/catalog/birdhouse/pcic/BCCAQv2/catalog.xml
```

Comment:
Shouldn't we use `pavics.ouranos.ca`?  `boreas` is an internal name so if Finch is deployed elsewhere than at Ouranos, it won't be accessible?  Or this is actually the intention?

Reply:
I wasn't aware of the distinction between pavics.ouranos.ca and boreas.ouranos.ca... Both worked, so I assumed they were an alias. I'll change it to pavics when I get the chance.

Reply:
Oh, both works from CRIM side?  I am a bit surprised, maybe @moulab88 has opened a special rule in the firewall for CRIM IP range.